elderly_assistant/services/speech.py

```
@@ -119,32 +119,43 @@ def _speak(self, text):
-                except Exception as e:
-                    self.last_error = f"edge-tts: {e}"
-                    self.logger.warning(f"edge-tts 播报失败，转 pyttsx3: {e}")
-            if self._pyttsx_engine:
-                try:
-                    self._pyttsx_engine.say(text)
-                    self._pyttsx_engine.runAndWait()
-                    return
-                except Exception as e:
-                    self.last_error = f"pyttsx3: {e}"
-                    self.logger.error(f"pyttsx3 播报失败: {e}")
+            # 进入播放：解除功放静音（非 M10 环境自动降级无操作）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)